<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForecastView</class>
 <widget class="QWidget" name="ForecastView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>888</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7" stretch="2,2,2,3,0,20">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="2,2,2,2,1,3,2,3">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLineEdit" name="xMin">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string>xMin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="xMax">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string>xMax</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yMin">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string>yMin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yMax">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string>yMax</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="savedLabel">
         <property name="styleSheet">
          <string notr="true">color: black;</string>
         </property>
         <property name="text">
          <string>Or</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="savedDropdown">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="placeholderText">
          <string>Select location...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timeLabel">
         <property name="styleSheet">
          <string notr="true">color: black;</string>
         </property>
         <property name="text">
          <string>Time</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="timeDropdown">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="placeholderText">
          <string>Select...</string>
         </property>
         <item>
          <property name="text">
           <string>0 hour</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4 hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6 hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12 hours</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,1,2">
       <item>
        <widget class="QPushButton" name="forecastVisualizeButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #2196F3;
color: white;</string>
         </property>
         <property name="text">
          <string>Visualize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="forecastSaveDataButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:disabled{background-color: #D9D9D9; color: white;} QPushButton:enabled{background-color: #2196F3; color: white;}</string>
         </property>
         <property name="text">
          <string>Save data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="forecastCompareButton">
         <property name="styleSheet">
          <string notr="true">background-color: #2196F3;
color: white;</string>
         </property>
         <property name="text">
          <string>Load saved data</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="forecastSaveSuccessNoti" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: #2521F3;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,1">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color: white;</string>
          </property>
          <property name="text">
           <string>Forecast data successfully saved!</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeNotiButton">
          <property name="styleSheet">
           <string notr="true">background-color: #2196f3; color: white;</string>
          </property>
          <property name="text">
           <string>Close this dialog</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="showLabel">
           <property name="styleSheet">
            <string notr="true">background-color: none;</string>
           </property>
           <property name="text">
            <string>Show</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="showAllRButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Show All</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="hideAllRButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Hide All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="2,2,3">
         <item row="0" column="0">
          <widget class="QCheckBox" name="roadTempCheckbox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Road temperature</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="windDirCheckbox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Wind direction</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="symbolCheckbox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Symbol</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="reliabilityCheckbox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Reliability</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="windSpeedCheckbox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Wind speed</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="reasonCheckbox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Forecast condition reason</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="airTempCheckBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Air temperature</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="daylightCheckbox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Daylight</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="roadCondCheckbox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Road condition</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="textPreview">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font-weight: bold;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter coordinates manually or choose pre-defined locations. &lt;/p&gt;&lt;p&gt;Click visualize to see the data&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="msgScrollArea">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>596</width>
          <height>1144</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QGridLayout" name="gridLayout_5" rowstretch="2,0,0,0,0,0">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <item row="3" column="0">
          <widget class="QLabel" name="forecastTitleLabel">
           <property name="styleSheet">
            <string notr="true">color: black;
font-weight: 500;
font-size: 20px;</string>
           </property>
           <property name="text">
            <string>Forecast data</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="saveWindButton">
           <property name="text">
            <string>Save wind graph as png</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="saveTempButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Save temperature graph as png</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="3">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QVBoxLayout" name="forecastScrollArea" stretch="0,0,0,0">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="currentCardFrame">
               <layout class="QVBoxLayout" name="currentCardLayout"/>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="compareCardFrame">
               <layout class="QVBoxLayout" name="compareCardLayout"/>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="airRoadFrame">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>500</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <layout class="QVBoxLayout" name="airRoadGraph"/>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="windSpeedFrame">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>500</height>
                </size>
               </property>
               <layout class="QVBoxLayout" name="windSpeedDirGraph"/>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
